mlflow/entities/trace.py

```diff
@@ -240,6 +243,52 @@ def _match_id(span: Span) -> bool:
             if _match_name(span) and _match_type(span) and _match_id(span)
         ]
 
+    def search_assessments(
+        self,
+        name: Optional[str] = None,
+        *,
+        span_id: Optional[str] = None,
+        all: bool = False,
```

Comment:
```suggestion
        include_overridden: bool = False,
```

weak suggestion: "all" sounds ok but might be a bit vague